daemon/api.go

```diff
@@ -1073,6 +1076,29 @@ func getChanges(c *Command, r *http.Request, user *auth.UserState) Response {
 		return BadRequest("select should be one of: all,in-progress,ready")
 	}
 
+	if wantedName := query.Get("for"); wantedName != "" {
+		selectFilter := filter
```

Comment:
Super nitpick, probably best to ignore. I stumbled a bit over the selectFilter name here and the filter arrangement. I wonder if something like "firstFilter" or "outerFilter" or "firstPassFilter" would have helped me here. But again, a tiny detail while reading the code and its probably just me.